gnocchiclient/exceptions.py

```diff
@@ -237,10 +239,11 @@ def from_response(response, method=None):
                 # Gnocchi json
                 desc = body.get('description')
                 if desc:
-                    for enhanced_cls in enhanced_classes:
-                        if enhanced_cls.match.match(desc):
-                            cls = enhanced_cls
-                            break
+                    if isinstance(desc, six.text_type):
```

Comment:
just add this check as part of previous `if desc`?

Reply:
done